### PR TITLE
Magento2 cmp custom page size via di

### PR DESCRIPTION
**Pull request by dairbuirabass**
Add instructions for custom page size via di

### CODE REVIEW

File: addons/cmp/guides/overriding-or-extending-functionalities/README.md [hunk 1]
```diff
@@ -99,3 +99,8 @@ A simple way to verify that the changes are working would be to log in as a cust
 
 Before using your custom module in production environment we strongly recommend that you follow the instructions how to package a Magento 2 module. If you are familiar with creating packages for composer, creating a package for M2 is a no-brainer. You can find the instructions from [here](http://devdocs.magento.com/guides/v2.0/extension-dev-guide/package/package_module.html)
 
+## Using custom value for category page size
```
Comment (olsi-qose) [inline addons/cmp/guides/overriding-or-extending-functionalities/README.md]: Better move this part into a separate page/file under this directory.

File: addons/cmp/guides/overriding-or-extending-functionalities/README.md [hunk 2]
```diff
@@ -99,3 +99,8 @@ A simple way to verify that the changes are working would be to log in as a cust
...
+
+Developers can set custom page size for category tagging via dependency injection. 
+
+Custom page size value can be passed for web handler `Nosto\Cmp\Plugin\Framework\Search\Request\WebHandler` and GraphQL handler `Nosto\Cmp\Plugin\Framework\Search\Request\GraphQlHandler`
```
Comment (olsi-qose) [inline addons/cmp/guides/overriding-or-extending-functionalities/README.md]: Please give the a code example of the required change e.x. https://docs.nosto.com/magento-2/addons/cmp/guides/overriding-or-extending-functionalities/customize-query-parameters#inject-the-customparameterresolver

File: addons/cmp/guides/overriding-or-extending-functionalities/README.md [hunk 3]
```diff
@@ -99,3 +99,8 @@ A simple way to verify that the changes are working would be to log in as a cust
 
 Before using your custom module in production environment we strongly recommend that you follow the instructions how to package a Magento 2 module. If you are familiar with creating packages for composer, creating a package for M2 is a no-brainer. You can find the instructions from [here](http://devdocs.magento.com/guides/v2.0/extension-dev-guide/package/package_module.html)
 
+## Using custom value for category page size
+
+Developers can set custom page size for category tagging via dependency injection.
```
Comment (olsi-qose) [inline addons/cmp/guides/overriding-or-extending-functionalities/README.md]: Change it to:

The CM module makes use of the default page size used in the storefront. There might be cases when the page size value might be incorrect. You can state the page size used by our module.

File: addons/cmp/guides/overriding-or-extending-functionalities/README.md (hunk 2 above)
Comment (olsi-qose) [inline addons/cmp/guides/overriding-or-extending-functionalities/README.md]: There are 3 use cases:

1. When MySQL is configured as the search engine. 
2. When ES is configured as the search engine
3. When the merchant is using M2 graphql to fetch the catalog page

**Review by olsi-qose — CHANGES_REQUESTED** [review]

File: addons/cmp/guides/overriding-or-extending-functionalities/README.md (hunk 2 above)
Comment (dairbuirabass) [reply addons/cmp/guides/overriding-or-extending-functionalities/README.md]: Done

File: addons/cmp/guides/overriding-or-extending-functionalities/README.md (hunk 3 above)
Comment (dairbuirabass) [reply addons/cmp/guides/overriding-or-extending-functionalities/README.md]: Done

File: addons/cmp/guides/overriding-or-extending-functionalities/README.md (hunk 2 above)
Comment (dairbuirabass) [reply addons/cmp/guides/overriding-or-extending-functionalities/README.md]: Done

**Review by olsi-qose — APPROVED** [review]

**Pull request merged**